[2024-03-28 17:52:08] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:08] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 1f79a851f8ffa3c5 127.0.0.1:58298 failed checkpoint posthandshake: Connected to self
[2024-03-28 17:52:08] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn static dial conn 1f79a851f8ffa3c5 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
[2024-03-28 17:52:13] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:13] [error] orderServer.go:593 p3Chain/core/blockOrdering.(*BlockOrderServer).simpleInit [Block Ordering] fail in simpleInit, no enough state messages collected!

[2024-03-28 17:52:18] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:18] [error] orderServer.go:593 p3Chain/core/blockOrdering.(*BlockOrderServer).simpleInit [Block Ordering] fail in simpleInit, no enough state messages collected!

[2024-03-28 17:52:23] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:28] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:33] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:38] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:39] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn inbound conn 1f79a851f8ffa3c5 127.0.0.1:58471 failed checkpoint posthandshake: Connected to self
[2024-03-28 17:52:39] [warning] server.go:631 p3Chain/p2p/server.(*Server).setupConn dyn dial conn 1f79a851f8ffa3c5 127.0.0.1:20130 failed checkpoint posthandshake: Connected to self
[2024-03-28 17:52:43] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:46] [warning] orderServer.go:506 p3Chain/core/blockOrdering.(*BlockOrderServer).tryCommit [Block Ordering] Not enough signatures got in round version 0000000000000000000000000000000000000000000000000000000000000000, height 0

[2024-03-28 17:52:48] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:53] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
[2024-03-28 17:52:58] [warning] separator.go:293 p3Chain/core/separator.(*Separator).DoBind Separator Construct: (1) nodes unable to establish P2P communication due to exceptions
